Packs/CiscoWebExFeed/Integrations/CiscoWebExFeed/CiscoWebExFeed.yml

```diff
@@ -82,34 +86,34 @@ configuration:
   name: insecure
   required: false
   type: 8
+  section: Connect
+  advanced: true
 - display: Use system proxy settings
   name: proxy
   required: false
   type: 8
-description: The WebEx IP Address and Domain web site provided by Cisco to document
-  IPs and Domains used by WebEx. The WebEx Feed integration fetches indicators from
-  the web page, with which you can create a list (allow list, block list, EDL, etc.)
-  for your SIEM or firewall service to ingest and apply to its policy rules.
+  section: Connect
+  advanced: true
+description: Use the Cisco WebEx Feed integration to fetch indicators from WeBex.
```

Comment:
Shouldn't this be 
Use the Cisco WebEx Feed integration to fetch indicators from WebEx.
or should it be Webex?  When I looked it up on line, the E is not uppercase.